.golangci.yml

```diff
@@ -9,7 +9,6 @@ linters-settings:
       - name: blank-imports
       - name: context-as-argument
       - name: context-keys-type
-      - name: dot-imports
```

Comment:
what is the reason for this change?

Reply:
All of our tests use dot imports for ginkgo and gomega.

Reply:
oh, so  removing `          only-new-issues: true` made all those files fail? and that's why it is necessary to disable?

Reply:
Yeah the action would create a patch just for the diff, and if nothing in the pull request specifically failed, then no error was returned. But the run on the main branch wouldn't do a diff, it would just scan the whole repo, finding other errors.